---
title: Methods
output:
  BiocStyle::html_document:
    code_folding: hide
    toc: yes
    toc_float: yes
---


```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```


RNA-Sequencing Analysis
Raw reads were quality checked with FastQC v0.11.7 (http://www.bioinformatics.babraham.ac.uk/projects/fastqc/).
Reads were aligned to the mouse reference genome (GRCm38.p6) using STAR v2.7.6a with default parameters ^[Dobin A, Davis CA, Schlesinger F, Drenkow J, Zaleski C, Jha S, Batut P, Chaisson M, Gingeras TR. STAR: ultrafast universal RNA-seq aligner. Bioinformatics. 2013 Jan 1;29(1):15-21. doi: 10.1093/bioinformatics/bts635. Epub 2012 Oct 25. PMID: 23104886; PMCID: PMC3530905.]. 
Gene abundances were calculated with featureCounts v2.0.1^[Liao Y, Smyth GK, Shi W. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. Bioinformatics. 2014 Apr 1;30(7):923-30. doi: 10.1093/bioinformatics/btt656. Epub 2013 Nov 13. PMID: 24227677.]  using composite gene models from Gencode release M25 ^[Harrow J, Frankish A, Gonzalez JM, Tapanari E, Diekhans M, Kokocinski F, Aken BL, Barrell D, Zadissa A, Searle S, Barnes I, Bignell A, Boychenko V, Hunt T, Kay M, Mukherjee G, Rajan J, Despacio-Reyes G, Saunders G, Steward C, Harte R, Lin M, Howald C, Tanzer A, Derrien T, Chrast J, Walters N, Balasubramanian S, Pei B, Tress M, Rodriguez JM, Ezkurdia I, van Baren J, Brent M, Haussler D, Kellis M, Valencia A, Reymond A, Gerstein M, Guigó R, Hubbard TJ. GENCODE: the reference human genome annotation for The ENCODE Project. Genome Res. 2012 Sep;22(9):1760-74. doi: 10.1101/gr.135350.111. PMID: 22955987; PMCID: PMC3431492.].
Differential expression analysis was performed in R using limma (v`r packageVersion("limma")`)^[Ritchie, M.E., Phipson, B., Wu, D., Hu, Y., Law, C.W., Shi, W., and Smyth, G.K. (2015). limma powers differential expression analyses for RNA-sequencing and microarray studies. Nucleic Acids Research 43(7), e47.], after removing lowly expressed genes  using the filterByExpr function from edgeR (v`r packageVersion("edgeR")`)^[Robinson MD, McCarthy DJ and Smyth GK (2010). edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 26, 139-140]. In brief, linear models were fitted with treatment information to create the design matrix, followed by empirical Bayes moderation of t-statistics. Raw P-values were adjusted for multiple testing using the Benjamini & Hochberg method ^[Benjamini, Y., and Hochberg, Y. (1995). Controlling the false discovery rate: a practical and powerful approach to multiple testing. Journal of the Royal Statistical Society Series B, 57, 289–300. doi: 10.1111/j.2517-6161.1995.tb02031.x. https://www.jstor.org/stable/2346101.], and only genes with an adjusted p < 0.10 were considered differentially expressed. Expression heatmaps were generated with pheatmap (v`r packageVersion("pheatmap")`)^[Raivo Kolde (2019). pheatmap: Pretty Heatmaps. R package version 1.0.12. https://CRAN.R-project.org/package=pheatmap] using log2 counts per million (CPM), with the values centered and scaled by row.
All scripts and code used for generating the bulk RNA-seq based figures can be found at https://github.com/abcwcm/Vartanian2023. 

# Session Info
```{r session, message=FALSE, warning=FALSE, cache=TRUE,comment="#",echo=FALSE, fig.width=10, fig.height=5.5, context="data"}
sessionInfo()
```

